### PR TITLE
Adding sub topic and event type

### PR DESCRIPTION
**Pull request by muhammadjunaidkhan-sap**
Adding sub topic and event type

### CODE REVIEW

File: src/api-reference/common/event-pubsub/travel-search-event.markdown [hunk 1]
```diff
@@ -69,16 +71,18 @@ Name|Type|Format|Description
 
 Name|Type|Format|Description
 ---|---|---|---
-`cityName`|`String`|YYYY-MM-DD|Name of the city .
-`cityIATACode`|`String`|YYYY-MM-DD|IATA code of the city.
+`cityName`|`String`|Frankfurt|Name of the city .
```
Comment (charlieconcur) [inline src/api-reference/common/event-pubsub/travel-search-event.markdown]: `Frankfurt` and `FRA` aren't formats, they are examples. Edit this as follows and you will be good:

```
`cityName`|`String`|-|Name of the city. Example: Frankfurt
`cityIATACode`|`String`|-|IATA code of the city. Example: FRA
```

File: src/api-reference/common/event-pubsub/travel-search-event.markdown [hunk 2]
```diff
@@ -69,16 +71,18 @@ Name|Type|Format|Description
...
+`cityIATACode`|`String`|FRA|IATA code of the city.
 
 ### <a name="schema-hotel-request"></a>Hotel
 
 Name|Type|Format|Description
 ---|---|---|---
 `id`|`String`|GUID|Uniquely identifies the event.
 `correlationId`|String|GUID|Uniquely identifies the hotel search request.
+`eventType`|String|travelSearchHotel|identifies the search event type.|
```
Comment (charlieconcur) [inline src/api-reference/common/event-pubsub/travel-search-event.markdown]: Formats which are other schemas should link to those schemas.

File: src/api-reference/common/event-pubsub/travel-search-event.markdown [hunk 3]
```diff
@@ -69,16 +71,18 @@ Name|Type|Format|Description
...
 `topic`|String|concur.travel.search|
+`subTopic`|String|hotelshop.v1.price|identify sub catagory for hotel search
```
Comment (charlieconcur) [inline src/api-reference/common/event-pubsub/travel-search-event.markdown]: The Format here is just confusing -- feel free to schedule some time with me so I can understand what you are trying to accomplish.

**Review by charlieconcur — CHANGES_REQUESTED** [review]
See the changes requested for edits needed.

File: src/api-reference/common/event-pubsub/travel-search-event.markdown (hunk 1 above)
Comment (muhammadjunaidkhan-sap) [reply src/api-reference/common/event-pubsub/travel-search-event.markdown]: @charlieconcurI have changed the document. please, do have a look. if you have more changes you want please let me know.

**Review by charlieconcur — APPROVED** [review]

**Pull request merged**